components/proxy/src/main/kotlin/com/hotels/styx/ObjectTags.kt

```diff
@@ -16,3 +16,10 @@
 package com.hotels.styx
 
 fun lbGroupTag(name: String) = "lbGroup=$name"
+
+fun matchLbGroupTag(tag: String): String? = "lbGroup=(.+)".toRegex()
```

Comment:
Maybe call this `lbGroupTagValue`, as that's what it is returning? (regex matching is just the mechanism).